docs/deployments/backing-services.md

```diff
@@ -340,11 +346,26 @@ The containers microservice component of this Reefer Container EDA reference app
 
 In order for the containers microservice to fire the BPM workflow, we need to provide the following information through Kubernetes configMaps and secrets:
 
-1. Provide the **BPM authentication login endpoint** and the **BPM workflow endpoint** in a configMap:
+1. Provide in a configMap:
+   * the **BPM authentication login endpoint**
+   * the **BPM workflow endpoint**
+   * the **BPM anomaly event threshold**
+   * the **BPM authentication token time expiration**
+
+You can _(and should)_ customize the values _(right-hand side)_ of each of the lines in the `data` section below to ensure your topics are only being published to by the correct deployments of microservices. Do not edit the keys _(left-hand side)_ of the any of the lines in the `data` section below, or else your microservice components will fail to start.
```

Comment:
This section doesn't really make sense here, as it's not as relevant as it is to topic definitions.  I would say it can be removed.

Reply:
agree